packages/loader/container-utils/src/error.ts

```diff
@@ -63,11 +62,10 @@ export class ThrottlingWarning extends LoggingError implements IThrottlingWarnin
     static wrap(
         error: unknown,
         retryAfterSeconds: number,
-        logger: ITelemetryLogger,
     ): IThrottlingWarning {
         const newErrorFn =
             (errMsg: string) => new ThrottlingWarning(errMsg, retryAfterSeconds);
-        return wrapErrorAndLog(error, newErrorFn, logger);
+        return wrapError(error, newErrorFn);
```

Comment:
Out of curiosity, what does wrap error do and what does wrap error and log do? Is it just a difference of logging?